virtio-devices/src/vhost_user/blk.rs

```diff
@@ -223,6 +223,12 @@ impl Drop for Blk {
                 error!("failed to kill vhost-user-blk: {:?}", e);
             }
         }
+        self.common.wait_for_epoll_threads();
```

Comment:
Looks like these vhost_user based functions are missing something like:
```rust
        self.common.epoll_threads = Some(epoll_threads);
```

in their activate

Reply:
I'm not sure if that's a win in usability/readability. See https://github.com/cloud-hypervisor/cloud-hypervisor/pull/5095#issuecomment-1380380561